salt/lax/bot-lax-adaptor.sls

```diff
@@ -119,19 +105,27 @@ dir-{{ path }}:
             - file: dir-{{ path }}
 {% endfor %}
 
-# added 2017-08-01 - temporary state, remove in due course
-old-log-files:
-    cmd.run:
-        - name: rm -f *.log
-        - cwd: /opt/bot-lax-adaptor
-        - user: {{ pillar.elife.deploy_user.username }}
-        - require:
-            - bot-lax-adaptor 
-
 bot-lax-writable-dirs:
     cmd.run:
         - name: echo "dirs created"
 
+#
+#
+#
+
+move /opt/bot-lax/article-xml/ to /ext/article-xml:
+    cmd.script:
+        - cwd: /ext
+        - source: salt://lax/scripts/mv-article-xml.sh
+        - onlyif:
```

Comment:
I think we can enable this on all stacks:
https://github.com/elifesciences/builder-private/blob/master/salt/top.sls#L126

Reply:
That file should create an `/ext` folder even if there's no volume.

Reply:
(on the default volume)

Reply:
PR: https://github.com/elifesciences/builder-private/pull/230

Reply:
Approved and merged, now `example.top` here should be updated along with losing the `onlyif` statements